src/components/filters/RangeFilter.vue

```diff
@@ -76,19 +100,29 @@ export default Vue.extend({
   },
   watch: {
     value (newValue) {
-      this.sliderValue[0] = Math.min(newValue[0], newValue[1])
-      this.sliderValue[1] = Math.max(newValue[0], newValue[1])
+      if (newValue[0] == null && newValue[1] == null) {
+        this.sliderValue = [null, null]
+      }
     }
   },
   methods: {
+    setSliderValue (id, value) {
+      this.sliderValue[id] = value
+      this.handleChange()
```

Comment:
handleSliderValueChange ? or just add to setSliderValue ?

Reply:
Improved naming